spec/expressions.md

```diff
@@ -663,48 +710,52 @@ F(10, 20, new object\[\] {});\
 F(10, 20, new object\[\] {30, 40});\
 F(10, 20, new object\[\] {1, "hello", 3.0});
```

Comment:
This block (starting on line 709, and the block starting on line 703 should be in "```csharp" blocks.